.github/workflows/ci.yml

```diff
@@ -131,6 +131,28 @@ jobs:
           fail_ci_if_error: false
           verbose: true
 
+  publish:
+    name: Publish crates on release
+    # Only do this job when publishing a release
+    if: github.event_name == 'release' && github.event.action == 'published'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - name: Bootstraping Grammars - Building
+        run: cargo build --package pest_bootstrap
+      - name: Bootstraping Grammars - Executing
+        run: cargo run --package pest_bootstrap
+      # it should figure out the right order: https://github.com/katyo/publish-crates#features
+      - name: Publish crates
+        uses: katyo/publish-crates@v1
+        with:
+          registry-token: ${{ secrets.CARGO_REGISTRY_TOKEN }}
```

Comment:
I forget whether publishing with the bootstrap info included requires `--allow-dirty`. I just know other crates doing similar things have required it in the past.